core/src/main/java/hudson/console/ConsoleNote.java

```diff
@@ -272,7 +272,7 @@ public static ConsoleNote readFrom(DataInputStream in) throws IOException, Class
 
             Jenkins jenkins = Jenkins.get();
             try (ObjectInputStream ois = new ObjectInputStreamEx(new GZIPInputStream(new ByteArrayInputStream(buf)),
-                    jenkins != null ? jenkins.pluginManager.uberClassLoader : ConsoleNote.class.getClassLoader(),
+                    jenkins.pluginManager.uberClassLoader,
```

Comment:
Nit: You can get rid of the local var in this case as well

Reply:
Thanks for pointing that out. Removed the local var also.